lib/galaxy/auth/__init__.py

```diff
@@ -64,8 +64,9 @@ def check_auto_registration(self, trans, login, password, no_password_check=Fals
                 if auth_results[0] is True:
                     try:
                         auth_return = parse_auth_results(trans, auth_results, options)
-                    except Conflict:
-                        break
+                    except Conflict as conflict:
+                        log.exception(conflict)
+                        raise
```

Comment:
If the `raise` here is meant to propagate the error to the user, you need to catch this exception at https://github.com/galaxyproject/galaxy/blob/dev/lib/galaxy/webapps/galaxy/controllers/user.py#L85

Reply:
The exception is now propagated to the user, i.e. converted to a message return by the autoregistration action inside of the user controller.